wpeverest/user-registration
# Tweak - Design changes in file upload of setting page

### All Submissions:






### Changes proposed in this Pull Request:
This PR changes the UI of file upload available in setting page like import forms, import users, import invite codes.

### How to test the changes in this Pull Request:

1. Go to Settings->Import/Export
2. check browse file design and test whether is working properly functionality wise too.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Design changes in file upload of setting page.
